.gitignore

```diff
@@ -9,10 +16,54 @@ build/
 !default.mode2v3
 *.perspectivev3
 !default.perspectivev3
+xcuserdata/
 *.xcworkspace
 !default.xcworkspace
-*xcuserdata
-profile
+
+## Other
 *.moved-aside
-DerivedData
+*.xccheckout
+*.xcscmblueprint
 .idea/
+profile
+.DS_Store
+
+## Obj-C/Swift specific
+*.hmap
+*.ipa
+*.dSYM.zip
+*.dSYM
+
+# CocoaPods
```

Comment:
This project doesn't have Cocoapods dependencies. I think we're getting ahead of ourselves by adding this line to our .gitignore